steps/mentor-calibre-lvs/configure.yml

```diff
@@ -71,6 +78,9 @@ parameters:
   #
   lvs_verify_netlist: 1
 
+  adk_view_lvs2: ""
+  lvs_extra_spice_include: ""
```

Comment:
We should only keep the "lvs_extra_spice_include" parameter here.

We should set that parameter in our graph directly (with .set_param) -- https://github.com/StanfordAHA/garnet/blob/master/mflowgen/Tile_PE/construct.py#L141